### ISSUE TITLE
Emulate Desktop View on non-responsive websites

### ISSUE THREAD

**Issue by alecos71** [fancyapps/fancybox#2398]
I would like **FancyBox** to have the feature of emulating the display view as if it were a Desktop on mobile devices when a global setting of the type is set:

`EmulateDesktopView = true;`

In this way even those who do not have a responsive website can exploit **FancyBox** without having to compromise excessively limiting or impeding.

Example:

```
$('[data-fancybox]').fancybox({
  EmulateDesktopView = true
});
```

What do you think? Will this feature ever see the light?

Thanks.

**Comment by fancyapps** [comment]
Hi,

Sorry, but I could not understand your question/comment. What do you mean by "Desktop on mobile devices"?

**Comment by alecos71** [comment]
Fancybox, on non responsive website, on mobile device doesn't work fine for me... but if I switch on desktop view option on mobile browser, fancybox displays correctly the images, so I did think that fancybox could have an option to run in desktop mode so the user could have the best experience on non responsive website. A sort on emulation for desktop view. So fancybox could run fine on non responsive website even if the user has a mobile device. I did think a super global setting that disables all fancybox mobile behaviors if set.

All in my humil opinion.

**Comment by fancyapps** [comment]
Hmm, could you explain this part - "on non responsive website, on mobile device doesn't work fine for me" - what does not work fine? Can you show screenshot or smth? I suspect this is related to page scaling, but I am not sure.

**Comment by alecos71** [comment]
Page scaling? What do you suggest as fancybox setting to test out this? Exists a special setting for this case? Thanks.

**Comment by fancyapps** [comment]
No, no, I was just guessing the possible cause of your issue. Sorry, I still have no idea what is your issue, any demo/screenshot would help.

**Comment by alecos71** [comment]
On non responsive website with a width of minimum 1200px, fancybox truncate the display of the overhead images and so cannot display anything while landscape images are truncated and is visible only a vertical portion of the full image.

See the attachment.

![Screenshot_20200109-181256_Firefox](https://user-images.githubusercontent.com/9997666/72090375-3cc00f80-330e-11ea-9409-3bfb66667f53.jpg)
![Screenshot_20200109-181321_Firefox](https://user-images.githubusercontent.com/9997666/72090402-49dcfe80-330e-11ea-9ff2-753a86343894.jpg)

**Comment by mjcampagna** [comment]
Why you’d make a non-responsive website in this day and age is quite beyond me. 

Nonetheless, have you tried setting a viewport meta tag on your document, if even to define the page as non-responsive?

**Comment by alecos71** [comment]
I did put on every page this meta:

`<meta name="viewport" content="width=device-width, initial-scale=1">`

My website non responsive works fine without fancybox. The only issue I have is with fancybox.

My health is not so good to permit to dedicate to make responsive website.

Sorry.

**Comment by alecos71** [comment]
Sorry, I solved by making a responsive css, now it's working fine... my website is now responsive.

Thanks for your patience. Have a nice day!
